code/modules/admin/verbs/custom_event.dm

```diff
@@ -38,3 +38,49 @@
 	to_chat(src, "<h2 class='alert'>A custom event is taking place. OOC Info:</h2>")
 	to_chat(src, "<span class='alert'>[html_encode(custom_event_msg)]</span>")
 	to_chat(src, "<br>")
+
+//admin event info to be view by admins
+
+/client/proc/cmd_admin_custom_event_info()
+	set category = "Event"
+	set name = "Change Custom Admin Event Info"
+
+	if(!holder && check_rights(R_EVENT))
```

Comment:
this can just be `if(!check_rights(R_EVENT)) to_chat(src, "Only administrators may use this command.") return`